openedx_lti_tool_plugin/views.py

```diff
@@ -300,15 +284,13 @@ def post(
             lineitem = ags_endpoint.get('lineitem')
 
             if not lineitem:
-                log.error('LTI AGS: Missing AGS lineitem.')
-                return HttpResponseBadRequest(self.BAD_RESPONSE_MESSAGE)
+                return LoggedHttpResponseBadRequest(_('LTI AGS: Missing AGS lineitem.'))
 
             if AGS_SCORE_SCOPE not in ags_endpoint.get('scope', []):
-                log.error('LTI AGS: Missing required AGS scope: %s', AGS_SCORE_SCOPE)
-                return HttpResponseBadRequest(self.BAD_RESPONSE_MESSAGE)
+                return LoggedHttpResponseBadRequest(_(f'LTI AGS: Missing required AGS scope: {AGS_SCORE_SCOPE}'))
 
             LtiGradedResource.objects.get_or_create(
-                lti_profile=lti_profile,
+                lti_profile=lti_profile[0],
```

Comment:
IndexError? @kuipumu

Reply:
@Squirrel18 I don't think is necessary since get_or_create_from_claims will always return a tuple